debian/rules

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/make -f
+
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
+PREPROCESS_FILES := $(wildcard debian/*.in)
+
+$(PREPROCESS_FILES:.in=): %: %.in
+	sed 's,/@DEB_HOST_MULTIARCH@,$(DEB_HOST_MULTIARCH:%=/%),g' $< > $@
+	chmod --reference=$< $@
```

Comment:
what is this for?

Reply:
to be honest, no clue :D Copied from the zigbee plugins rules, which in turn I believe was copied from the nymea-plugins rules and thinking about that, it's likely to do with the postinst.in scripts being executable...